jerry-core/mem/mem-heap.cpp

```diff
@@ -349,7 +349,7 @@ mem_heap_init (void)
   for (size_t i = 0; i < MEM_HEAP_CHUNKS_NUM; i++)
   {
 #ifndef JERRY_NDEBUG
-    JERRY_ASSERT (mem_heap_length_types[i] == mem_block_length_type_t::GENERAL);
+    JERRY_ASSERT (mem_heap_length_types[i] == GENERAL);
```

Comment:
@jiangzidong, please rename `GENERAL` / `ONE_CHUNKED` to `MEM_BLOCK_LENGTH_TYPE_GENERAL` / `MEM_BLOCK_LENGTH_TYPE_ONE_CHUNKED`.